mgit.go

```diff
@@ -49,18 +49,21 @@ func main() {
 					whitelist := strings.FieldsFunc(c.String("whitelist"), func(c rune) bool {
 						return c == ','
 					})
-					return provider.Clone(whitelist, c.String("has-file"))
+					return provider.Clone(whitelist, c.String("has-file"), c.String("content-regexp"))
 				},
 				Flags: []cli.Flag{
 					&cli.StringFlag{
 						Name:  "has-file",
 						Usage: "only clone repo which has file",
 					},
-
 					&cli.StringFlag{
 						Name:  "whitelist",
 						Usage: "only clone repos in comma separated list",
 					},
+					&cli.StringFlag{
+						Name:  "content-regexp",
+						Usage: "only clone repos in comma separated list",
```

Comment:
Hmm should this not be something like `Checks this regexp inside the has-file` or better